src/HPWH.cc

```diff
@@ -65,6 +65,58 @@ const double HPWH::MAXOUTLET_R410A = F_TO_C(140.);
 
+HPWH::DrawPattern HPWH::verySmallUsage = {
+    {HM_TO_MIN(0, 00), 7.6, 3.8},
+    {HM_TO_MIN(1, 00), 3.8, 3.8},
+    {HM_TO_MIN(1, 05), 1.9, 3.8},
+    {HM_TO_MIN(1, 10), 1.9, 3.8},
+    {HM_TO_MIN(1, 15), 1.9, 3.8},
+    {HM_TO_MIN(8, 00), 3.8, 3.8},
+    {HM_TO_MIN(8, 15), 7.6, 3.8},
+    {HM_TO_MIN(9, 00), 5.7, 3.8},
+    {HM_TO_MIN(9, 15), 3.8, 3.8},
+};
+
+HPWH::DrawPattern HPWH::lowUsage = {{HM_TO_MIN(0, 00), 56.8, 6.4},
+                                    {HM_TO_MIN(0, 30), 7.6, 3.8},
+                                    {HM_TO_MIN(1, 00), 3.8, 3.8},
+                                    {HM_TO_MIN(10, 30), 22.7, 6.4},
+                                    {HM_TO_MIN(11, 30), 15.1, 6.4},
+                                    {HM_TO_MIN(12, 00), 3.8, 3.8},
+                                    {HM_TO_MIN(12, 45), 3.8, 3.8},
+                                    {HM_TO_MIN(12, 50), 3.8, 3.8},
+                                    {HM_TO_MIN(16, 15), 7.6, 3.8},
+                                    {HM_TO_MIN(16, 45), 7.6, 6.4},
+                                    {HM_TO_MIN(17, 00), 11.4, 6.4}};
+
+HPWH::DrawPattern HPWH::mediumUsage = {{HM_TO_MIN(0, 00), 56.8, 6.4},
+                                       {HM_TO_MIN(0, 30), 7.6, 3.8},
+                                       {HM_TO_MIN(1, 40), 34.1, 6.4},
+                                       {HM_TO_MIN(10, 30), 34.1, 6.4},
+                                       {HM_TO_MIN(11, 30), 18.9, 6.4},
+                                       {HM_TO_MIN(12, 00), 3.8, 3.8},
+                                       {HM_TO_MIN(12, 45), 3.8, 3.8},
+                                       {HM_TO_MIN(12, 50), 3.8, 3.8},
+                                       {HM_TO_MIN(16, 00), 3.8, 3.8},
+                                       {HM_TO_MIN(16, 15), 7.6, 3.8},
+                                       {HM_TO_MIN(16, 45), 7.6, 6.4},
+                                       {HM_TO_MIN(17, 00), 26.5, 6.4}};
+
+HPWH::DrawPattern HPWH::highUsage = {{HM_TO_MIN(0, 00), 102, 11.4},
+                                     {HM_TO_MIN(0, 30), 7.6, 3.8},
+                                     {HM_TO_MIN(0, 40), 3.8, 3.8},
+                                     {HM_TO_MIN(1, 40), 34.1, 6.4},
+                                     {HM_TO_MIN(10, 30), 56.8, 11.4},
+                                     {HM_TO_MIN(11, 30), 18.9, 6.4},
+                                     {HM_TO_MIN(12, 00), 3.8, 3.8},
+                                     {HM_TO_MIN(12, 45), 3.8, 3.8},
+                                     {HM_TO_MIN(12, 50), 3.8, 3.8},
+                                     {HM_TO_MIN(16, 00), 7.6, 3.8},
+                                     {HM_TO_MIN(16, 15), 7.6, 3.8},
+                                     {HM_TO_MIN(16, 30), 7.6, 6.4},
+                                     {HM_TO_MIN(16, 45), 7.6, 6.4},
+                                     {HM_TO_MIN(17, 00), 53.0, 11.4}};
+
```

Comment:
Do we want to make this all a single `std::unordered_map<Usage,DrawPattern>` to avoid if statements elsewhere in the code?